index.js

```diff
@@ -0,0 +1,251 @@
+const app = () => {
+  switch (document.body.id) {
+    case "searchStock":
+      displaySearchPage();
+      break;
+    case "stockInfo":
+      displayCompanyPage();
+      break;
+  }
+
+  function displaySearchPage() {
+    async function fetchMarqueeData() {
+      try {
+        let url = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/quotes/nyse?exchange=NASDAQ`;
+        let response = await fetch(url);
+        let data = await response.json();
+        let marqueeContainer = document.getElementById("marquee__container");
+        for (let j = 0; j < 4; j++) {
+          for (let i = 0; i < 10; i++) {
+            let marqueePrice = document.createElement("div");
+            marqueePrice.className = "marquee-price";
+            let marqueeSymbol = document.createElement("div");
+            marqueeSymbol.className = "marquee-name";
+            const symbol = data[i].symbol;
+            const price = `${data[i].price}$`;
+            marqueeSymbol.innerHTML = symbol;
+            marqueePrice.innerHTML = price;
+            marqueePrice.classList.add("green");
+            marqueeContainer.appendChild(marqueeSymbol);
+            marqueeContainer.appendChild(marqueePrice);
+          }
+        }
+      } catch (err) {
+        console.log(err);
```

Comment:
you should add error handling code here. Let the end user know that something went wrong and consider offering solution (try agian? refresh the page?)